### PR TITLE
Frontend np percentile

### LINKED ISSUE #10534: percentile
This issue aims to solve the subtask "percentile" from the ToDo List.

### LINKED ISSUE #13405: percentile

### PR DESCRIPTION
**Pull request by pradyyadav**
close #13405

### CODE REVIEW

**Comment by pradyyadav** [comment]
dependent on PR  #13401

File: ivy/functional/frontends/numpy/statistics/order_statistics.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    a = ivy.array(a)
+
+    ret = ivy.percentile(a, q, axis=axis, overwrite_input=overwrite_input,
+                         method=method, keep_dims=keepdims, 
+                         interpolation=interpolation, out=out)
+
+    return ret
```
Comment (Daniel4078) [inline ivy/functional/frontends/numpy/statistics/order_statistics.py:31]: please do not forget to have one empty line at the end of each files!

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello! I know that we can not check the actural behavior of your frontend implementation. please for now just fix the small formatting error below.

**Comment by pradyyadav** [comment]
@Daniel4078, thank you for your review, but as per ivy documentation, the frontends are supposed to follow their respective docs for easy to understand and refactor. I see that percentile is implemented at order statistics, please find the link: https://numpy.org/doc/stable/reference/routines.statistics.html. Please correct me if I got something wrong.

**Comment by Daniel4078** [comment]
> @Daniel4078, thank you for your review, but as per ivy documentation, the frontends are supposed to follow their respective docs for easy to understand and refactor. I see that percentile is implemented at order statistics, please find the link: https://numpy.org/doc/stable/reference/routines.statistics.html. Please correct me if I got something wrong.

you are very correct! then can you also try check if there is any other Order statistics function that is placed in the wrong place? Thanks!

**Comment by pradyyadav** [comment]
> > @Daniel4078, thank you for your review, but as per ivy documentation, the frontends are supposed to follow their respective docs for easy to understand and refactor. I see that percentile is implemented at order statistics, please find the link: https://numpy.org/doc/stable/reference/routines.statistics.html. Please correct me if I got something wrong.
> 
> you are very correct! then can you also try check if there is any other Order statistics function that is placed in the wrong place? Thanks!

Sure I would be happy to do it. However Daniel, I need some help with the dependent PR. Would you like to voluntarily review it?

**Comment by Daniel4078** [comment]
I just cleared the current assignee for that pull request of yours, the bot will soon get you a new assignee that is more responsive I hope.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Daniel4078** [comment]
@pradyyadav How is your other pull request going?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**